setup.py

```diff
@@ -31,6 +31,7 @@
     "sphinx",
     "sphinx-click",
     "sphinx-copybutton",
+    "sphinx-tabs",
```

Comment:
I got this error:

```
(dsgrid) C:\projects\dsgrid\docs>make.bat html
The system cannot find the file specified.
Running Sphinx v5.1.1

Exception occurred:
  File "C:\Users\ehale\Anaconda3\envs\dsgrid\lib\site-packages\sphinx_tabs\tabs.py", line 341, in setup
    app.add_javascript(path)
AttributeError: 'Sphinx' object has no attribute 'add_javascript'
The full traceback has been saved in C:\Users\ehale\AppData\Local\Temp\1\sphinx-err-eggiykhw.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```

so I tried upgrading Sphinx, which changed my Sphinx version from 5.1.1 to 7.2.2, but that didn't fix the problem:

```
(dsgrid) C:\projects\dsgrid\docs>make.bat html
The system cannot find the file specified.
Running Sphinx v7.2.2
C:\Users\ehale\Anaconda3\envs\dsgrid\lib\site-packages\sphinxcontrib\autodoc_pydantic\directives\autodocumenters.py:20: RemovedInSphinx80Warning: The alias 'sphinx.util.typing.stringify' is deprecated, use 'sphinx.util.typing.stringify_annotation' instead. Check CHANGES for Sphinx API modifications.
  from sphinx.util.typing import get_type_hints, stringify

Exception occurred:
  File "C:\Users\ehale\Anaconda3\envs\dsgrid\lib\site-packages\sphinx_tabs\tabs.py", line 341, in setup
    app.add_javascript(path)
AttributeError: 'Sphinx' object has no attribute 'add_javascript'
The full traceback has been saved in C:\Users\ehale\AppData\Local\Temp\1\sphinx-err-2u3wj4vb.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```

Reply:
I think the culprit is this extension: `sphinx.ext.autosectionlabel`. I’m not sure if we’re still using it. I have found it to be occasionally useful, but it often causes this problem.